pkg/specgen/generate/oci.go

```diff
@@ -165,7 +165,7 @@ func SpecGenToOCI(ctx context.Context, s *specgen.SpecGenerator, rt *libpod.Runt
 			inUserNS = true
 		}
 	}
-	if inUserNS && s.NetNS.IsHost() {
+	if inUserNS && s.NetNS.NSMode != specgen.NoNetwork {
```

Comment:
doesn't it work when the NSMode is set to private?

Reply:
I suspect it doesn't because we still make the namespace in Podman in that case

Reply:
I'll check again to verify.

Reply:
You are entirely correct, it does work.